### DISCUSSION TITLE
What am I doing wrong? Tabout doesn't work..

### DISCUSSION THREAD

**Discussion by doyeonx** [abecodes/tabout.nvim#55] [Q&A]
The tabout extension doesn't seem to be working at all...

https://github.com/abecodes/tabout.nvim/assets/115525965/e961540e-20e4-49a5-99d2-915702cfe2ba

I included tabout in my plugins-setup.lua:
```
use({
  "abecodes/tabout.nvim",
  branch = "feature/tabout-md",
  wants = { "nvim-treesitter" },
  after = { "nvim-cmp" },
})
```
I also made a config file (copied from the docs):
```
local setup, tabout = pcall(require, "tabout")
if not setup then
  return
end

tabout.setup({
  tabkey = "<Tab>",
  backwards_tabkey = "<S-Tab>",
  act_as_tab = true,
  act_as_shift_tab = true,
  default_tab = "<C-t>",
  default_shift_tab = "<C-d>",
  enable_backwards = true,
  completion = true,
  tabouts = {
    { open = "'", close = "'" },
    { open = '"', close = '"' },
    { open = "`", close = "`" },
    { open = "(", close = ")" },
    { open = "[", close = "]" },
    { open = "{", close = "}" },
  },
  ignore_beginning = true, --[[ if the cursor is at the beginning of a filled element it will rather tab out than shift the content ]]
  exclude = {},
})
```

But it doesn't seem to work on any of the files. I did include it in my init.lua file as well (require("doyeon.plugins.tabout")) and I checked multiple issues and discussions by other and tried their solutions. None of them seemed to work. Am I doing something wrong?

**Comment by abecodes** [comment]
Hi @KJ3RO 😊 

Maybe if you use the master branch?

Can you check `:imap <TAB>`? If you see smth with `<Plug>(Tabout)` everything is set up correctly.

**Reply by doyeonx** [reply]
Hi, thanks for your reply.
I see
```
<Tab>       * <Lua 562: ~/.local/share/nvim/site/pack/packer/start/nvim-cmp/lua/cmp/utils/keymap.lua:127>
                 cmp.utils.keymap.set_map
```
Is this right or does this mean that I've done something wrong?

**Reply by abecodes** [reply]
That is not what it should look like.
Seems like it is overwritten by smth related to cmp

**Reply by doyeonx** [reply]
Okay, Thanks!
